test/views/ConfirmSubmissionViewSpec.scala

```diff
@@ -296,5 +297,73 @@ class ConfirmSubmissionViewSpec extends ViewBaseSpec {
         }
       }
     }
+
+    "user is non-agent on the confirmSubmission page with the bta links partial" should {
```

Comment:
You're already testing this content in BtaNavigationLinksTemplateSpec.scala so in this spec, you should only test that passing some HTML as a parameter then puts that HTML on the page